pages/account/manage/event/[[...data]].js

```diff
@@ -257,6 +259,14 @@ export default function ManageEvent({ BASE_URL, event }) {
                       setEnabled={setIsVirtual}
                     />
                   </div>
+                  <div className="mt-1 sm:col-span-2 sm:mt-0">
+                    <Input
+                      name="color"
+                      label="Color"
+                      onChange={(e) => setColor(e.target.value)}
+                      value={color}
```

Comment:
The model has a `min` and `max`, this should be added here for client size validation

Reply:
Ahh yes